app/models/alert.rb

```diff
@@ -77,6 +77,12 @@ def email_target_user
   def emails_disabled?
     ENV["#{self.class}_emails_disabled"] == 'true'
   end
+
+  def resolvable?
```

Comment:
Instead of type checking in `Alert`, this should return false, and 'true' version (or `!resolved`) should be in the `ArticlesForDeletionAlert` subclass to override it.